example/launcher_cr.yaml

```diff
@@ -6,7 +6,44 @@ spec:
 
 ####### OpenShift Configuration
   openshift:
+    apiUrl: https://openshift.default.svc.cluster.local
     consoleUrl: #<the OpenShift Console Url (ex: 'https://192.168.64.4:8443')>#
+    clusters:
```

Comment:
maybe remove this, so that we keep the example as small as possible?

Reply:
This has now been removed from the Launcher custom resource example :)